app/services/PayeAtsService.scala

```diff
@@ -48,4 +56,16 @@ trait PayeAtsService {
        }
      }
   }
+
+  private def sendAuditEvent(taxYear: Int)(implicit hc: HeaderCarrier, request: PayeAuthenticatedRequest[_]) = {
+    auditService.sendEvent(
+      auditType = AuditTypes.Tx_SUCCEEDED,
+      details = Map(
+        "userId" -> request.userId,
+        "userNino" -> request.nino.nino,
```

Comment:
This nino could not be the same nino being passed in to `middleConnector.connectToPayeATS(nino`

Reply:
We are passing request.nino from controller that is being passed to the connector.
So both nino are same but i understand the confusion will update the code accordingly.

Reply:
I agree their is a risk that this isn't the NINO we used (even though the call to the backend should fail if they don't match). I would agree with changing and using the one from the call.